src/php/ext/grpc/call_credentials.c

```diff
@@ -209,14 +209,6 @@ void plugin_get_metadata(void *ptr, grpc_auth_metadata_context context,
 
 /* Cleanup function for plugin creds API */
 void plugin_destroy_state(void *ptr) {
-  plugin_state *state = (plugin_state *)ptr;
```

Comment:
Why is this being deleted?  We should avoid calling the destructor util we are ready instead.